### PR TITLE
Organization View - closes #19

### LINKED ISSUE #19: Create organization view
_From @bajiat on February 19, 2016 12:14_

Create initial markup for public organization view. Include i18n tags.

Definition of done
---
* [x] Basic organization markup exists, optionally using CSS framework semantics.
* [ ] i18n tags have been included for any UI texts.
* [x] Subscription to Organization publication exists.

_Copied from original issue: Dipor/platform#7_

# Reference
Please see the [requirements document (organization view section)](https://github.com/Digipalvelutehdas/dipor-dashboard/blob/master/docs/requirements.md#organization-view-registered-user-who-is-a-member-of-the-organization-in-question) for our current understanding of the organization view.

# Depends on
* #9
* #13 
* #4

### PR DESCRIPTION
**Pull request by frenchbread**
Closes #19 

# Proposed changes

- Add organization view template
- Add organization fields

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by bajiat** [comment]
@frenchbread Is this WIP or ready for review?

**Comment by frenchbread** [comment]
@bajiat It's WIP

**Comment by frenchbread** [comment]
This PR is not longer WIP. But since it contains commits that are not a part of this issue, we might need to wait until they are merged.

**Comment by frenchbread** [comment]
Related PR has been merged.

Please review this.

File: organizations/client/single-organization-view/single-organization-view.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    instance.organizationId = FlowRouter.current().params.organizationId;
+
+    // Subscribe to singleOrganization publication and pass organization Id
+    instance.subscribe('singleOrganization', instance.organizationId);
+
+};
+
+Template.singleOrganizationView.helpers({
+    singleOrganization: function () {
+
```
Comment (brylie) [inline organizations/client/single-organization-view/single-organization-view.js]: Missing line comment

File: organizations/client/single-organization-view/single-organization-view.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        var instance = Template.instance();
+
+        return Organizations.findOne(instance.organizationId);
```
Comment (brylie) [inline organizations/client/single-organization-view/single-organization-view.js]: Missing line comment

File: organizations/client/single-organization-view/single-organization-view.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<template name="singleOrganizationView">
+
+    <div class="container">
+        <div>
+            <h1 >{{singleOrganization.name}}</h1>
+            <p><i >{{singleOrganization.description}}</i></p>
```
Comment (brylie) [inline organizations/client/single-organization-view/single-organization-view.html]: Check whitespace in HTML tag.

**Comment by brylie** [comment]
This looks good. There are just a couple of issues regarding code documentation and lint.

**Comment by frenchbread** [comment]
Added suggested changes & resolved conflicts.

**Comment by jykae** [comment]
Ok, great merged already.

**Pull request merged**